Bottom: 5b3e6b0c8f55db890054197f046be33a076ad8c8
Top:    9d6859d28318ebe2af5e13a6a49f85b73cadfa86
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 01:44:42 +0100

Refresh of drop-in-pcb-gl-code-various-mess

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index cf44922..1431d93 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -26,7 +26,6 @@
 #include <dmalloc.h>
 #endif
 
-
 RCSID ("$Id$");
